cmd/captplanet/run.go

```diff
@@ -47,7 +47,7 @@ var (
 	runCfg struct {
 		HeavyClient HeavyClient
 		Farmers     [farmerCount]Farmer
-		Gateway     miniogw.Config
+		Uplink      uplink.Config
```

Comment:
yeah, this field rename i think is right, even if the package name stays the same